spec/lib/server_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rack/test"
+
+describe Server::Server do
+  include Rack::Test::Methods
+
+  def app
+    Server::Server
+  end
+
+  context "GET /api/v1/forms" do
+    it "returns a status code 401 when no authentication supplied" do
+      get "/api/v1/forms"
+      expect(last_response.status).to eq(401) unless ENV["API_KEY"].nil?
```

Comment:
I was having trouble dealing with the case where the API_KEY env hasn't been set - I've just bypassed he first and last tests if the env hasn't been set. I feel like this isn't the best way to do this, so please let me know if you have any better ideas

Reply:
hmmmmmmmmmm I think we should mock this in the tests?

Reply:
yeah good point

Reply:
ok I've stubbed the api key env var now.